### DISCUSSION TITLE
Untitled

### DISCUSSION THREAD

**Discussion by ghost** [quiple/galmuri#29] [Ideas]
***(Sorry, I cannot speak Korean so I wrote it in English)***

Hello, My team is making a Simplified Chinese version of this font.
The font will be released after my senior high school entrance examination.

**Comment by quiple** [comment]
Thanks for contribute! Perhaps the time has come to consider Chinese language support...

BTW, What do you think about combining [Ark Pixel Font](https://github.com/TakWolf/ark-pixel-font) or [Cubic 11](https://github.com/ACh-K/Cubic-11) or [Chill-Bitmap](https://github.com/Warren2060/Chill-Bitmap), by any chance?

**Reply by ghost** [reply]
But Chill Bitmap's license is not OFL

**Reply by quiple** [reply]
> But Chill Bitmap's license is not OFL

I thought it was a compatible license. If not, I'll exclude it.

**Reply by ghost** [reply]
It is also a "Share-alike" license

**Reply by quiple** [reply]
> It says “THESE FONTS ARE PROVIDED "AS IS" WITHOUT WARRANTY.”

That's just disclaimer. The same is included in the OFL.
<https://scripts.sil.org/cms/scripts/page.php?item_id=OFL_web#c6abc7e0>

You can find out more about this on [Wikipedia](https://en.wikipedia.org/wiki/As_is).

**Reply by ghost** [reply]
It says “本字型的衍生品之授权必须与此字型相同”

>Derivatives of this font must be authorized to be the same as this font.
>(Translated on Bing)

**Reply by quiple** [reply]
Okay, I see. How about the other fonts?

**Reply by ghost** [reply]
I'm afraid combining other fonts isn't a good idea.

**Reply by quiple** [reply]
Why do you think so? Just so you know, this font project started with bitmap font design of Nintendo DS, not entirely new.

And wouldn't it take a long time to add a lot of Chinese character glyphs?

**Reply by ghost** [reply]
Yes.

**Reply by ghost** [reply]
> Why do you think so?

Because their styles are different from ours.

**Reply by quiple** [reply]
Oh, I didn't realize it because I'm not used to Chinese characters because I'm Korean. Thanks for advice!
